charts/gha-runner-scale-set/templates/_helpers.tpl

```diff
@@ -229,8 +246,20 @@ env:
   - name: RUNNER_WAIT_FOR_DOCKER_IN_SECONDS
     value: "120"
     {{- end }}
+    {{- if $setNodeExtraCaCerts }}
+  - name: NODE_EXTRA_CA_CERTS
+    value: {{ clean (print $tlsConfig.runnerMountPath "/" $tlsConfig.certificateFrom.configMapKeyRef.key) }}
```

Comment:
do we need to add `"` around the string in case the path contains space?

Reply:
We should put quotes around names as well, because filename can also contain spaces

Reply:
Resolving all these as spaces are not allowed in config map keys (must satisfy `'[-._a-zA-Z0-9]+'`).